pyanaconda/modules/storage/bootloader/bootloader_interface.py

```diff
@@ -232,3 +234,22 @@ def IsPasswordSet(self) -> Bool:
     def IsPasswordEncrypted(self) -> Bool:
         """Is the GRUB boot loader password encrypted?"""
         return self.implementation.password_is_encrypted
+
+    @property
+    def SecureBoot(self) -> Int:
+        """The Secure Boot mode."""
+        return self.implementation.secure_boot.value
+
+    @emits_properties_changed
+    def SetSecureBoot(self, value: Int):
+        """Set up the Secure Boot mode.
+
+        Supported values:
+           -1  Default.
+            0  Disable Secure Boot.
+            1  Enable Secure Boot unconditionally.
```

Comment:
What happens if Secure Boot is unconditionally enabled yet hardware does not support it ? I guess the installation will fail ? If so, that should be documented somewhere.

Reply:
Yes, it will fail. The docstring is updated.